gpflow/kernels/base.py

```diff
@@ -167,8 +167,8 @@ class Combination(Kernel):
 
     _reduction = None
 
-    def __init__(self, kernels: List[Kernel], name: Optional[str] = None):
-        super().__init__(name=name)
+    def __init__(self, kernels: List[Kernel], name: Optional[str] = None, **kwargs):
+        super().__init__(name=name, **kwargs)
```

Comment:
If you're already just passing through, might as well drop the `name` argument (I don't think anyone should be setting that positionally anyways).

This raises an issue with documentation - how will end users know what the accepted arguments are in the end? Sphinx would have to parse the `__init__` of every single parent class. Does it actually do that?

Reply:
Documentationwise, it doesnt document the all the args. The concrete 'leaf/child' cases should know whats necessary and pass to all parents... Would that work?

Reply:
```suggestion
        super().__init__(**kwargs)
```